draftjs_exporter/style_state.py

```diff
@@ -57,12 +58,19 @@ def get_style_value(self):
         return ''.join(sorted(rules))
 
     def add_node(self, element, text):
+
+        for deco in self.composite_decorators:
+            text = deco.process(text, parent=element)
```

Comment:
Oh this is really a tricky problem! Even if we only replace in the text rather than the HTML, some nested decorators would still be problematic. In your example, if the hashtag decorator replaces the `#test` with a link, of course we can avoid replacing the first one because it's not in the text, but the second one should not be replaced either.

I'll look into this, as soon as I got a little time.